test/lib/router.js

```diff
@@ -1421,6 +1421,26 @@ describe('Router', function () {
         done();
       });
     });
+
+    it('responds with 404 when has a trailing slash', function (done) {
+      var app = new Koa();
+      var router = new Router({
+        sensitive: true
+      });
+      router.get('/info', function (ctx) {
+          ctx.body = 'hello';
+      });
+      request(http.createServer(
```

Comment:
This can probably be a one-liner like the rest of the test suite. Makes it easier to search / replace when its consistent.